tests/models/vits/test_modeling_vits.py

```diff
@@ -168,30 +181,30 @@ def test_model_forward(self):
     def test_determinism(self):
         pass
 
-    # TODO: Fix me (ydshieh)
-    @unittest.skip("currently failing")
+    @is_flaky(description="Weight initialisation for the VITS conv layers sometimes exceeds the kaiming normal range")
```

Comment:
let's put `max_attempts=3` here. The default is `5`. Not a big deal but just my preference.

Reply:
Good for me! Resolved in https://github.com/huggingface/transformers/pull/25945/commits/4f970d273b390c729be07b435134af8f90e4ba01